lcof2/剑指 Offer II 027. 回文链表/README.md

```diff
@@ -349,6 +349,55 @@ public class Solution {
 }
 ```
 
+#### Swift
+
+```swift
+/**
+ * Definition for singly-linked list.
+ * public class ListNode {
+ *     var val: Int
+ *     var next: ListNode?
+ *     init() { self.val = 0; self.next = nil; }
+ *     init(_ val: Int) { self.val = val; self.next = nil; }
+ *     init(_ val: Int, _ next: ListNode?) { self.val = val; self.next = next; }
+ * }
+ */
+
+class Solution {
+    func isPalindrome(_ head: ListNode?) -> Bool {
+        guard let head = head else { return true }
+        
+        var slow = head
+        var fast = head.next
+        while fast != nil && fast?.next != nil {
+            slow = slow.next!
+            fast = fast?.next?.next
+        }
+        
+        var cur = slow.next
+        slow.next = nil
```

Comment:
Hello, this line of code needs to be removed, otherwise the program will not pass the test cases.

Reply:
line 378